### PR TITLE
init files

### PR DESCRIPTION
**Pull request by yonglbl**
init files

### CODE REVIEW

File: BRAVE/backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+.pybuilder/
+target/
+
+# Jupyter Notebook
+.ipynb_checkpoints
```
Comment (oruebel) [inline BRAVE/backend/.gitignore:79]: I think the `.ipynb_checkpoints` folder that is part of this PR should probably be removed before merging the PR

Comment (yonglbl) [reply BRAVE/backend/.gitignore:79]: ipynb_checkpoints directory removed

**Review by oruebel — APPROVED** [review]

File: BRAVE/frontend/src/jbrowse/defaultSessions/NC_003899_1.js [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+        hideHeaderOverview: false,
+        hideNoTracksActive: false,
+        trackSelectorType: 'hierarchical',
+        trackLabels: 'overlapping',
+        showCenterLine: false,
+        showCytobandsSetting: true,
+        showGridlines: true,
+    },
+};
+
```
Comment (ajtritt) [inline BRAVE/frontend/src/jbrowse/defaultSessions/NC_003899_1.js:269]: Can you delete the lines below here?

File: BRAVE/frontend/src/jbrowse/defaultSessions/NC_003899_1.js [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
```
Comment (ajtritt) [inline BRAVE/frontend/src/jbrowse/defaultSessions/NC_003899_1.js:1]: Do we need one of these for every genome?

**Review by ajtritt — APPROVED** [review]
Looks good to me. Will we will need a `defaultSession` and `tracks` for every genome?

**Pull request closed without merging**